pkgs/by-name/ne/netatalk/0000-no-install-under-usr-cupsd.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/config/meson.build b/config/meson.build
+index 4d750dba..f6bd6e6e 100644
+--- a/config/meson.build
++++ b/config/meson.build
+@@ -55,15 +55,15 @@ elif host_os in ['linux']
+     cups_libdir = '/usr/lib'
+ endif
+ 
+-if have_appletalk and have_cups and cups_libdir != ''
+-    configure_file(
+-        input: 'pap.in',
+-        output: 'pap',
+-        configuration: cdata,
+-        install: true,
+-        install_dir: cups_libdir / 'cups/backend',
+-    )
+-endif
++#if have_appletalk and have_cups and cups_libdir != ''
++#    configure_file(
++#        input: 'pap.in',
++#        output: 'pap',
++#        configuration: cdata,
++#        install: true,
++#        install_dir: cups_libdir / 'cups/backend',
++#    )
++#endif
```

Comment:
Is the standard CUPS libdir not writable on Nix? Or is Nix storing CUPS backends in a different dir?

Reply:
yep. Nix/NixOS is not FHS compliant. Everything is `/nix/store`-centric.